_docs/howto/sicherheitstipps.md

```diff
@@ -15,12 +15,12 @@ HTTP Strict Transport Security (HSTS) ist ein Webserver Verzeichnis, das Benutze
 Damit wird der `Strict-Transport-Security`-Parameter festgelegt. Es zwingt diese Verbindungen zur HTTPS Verschlüsselung, und ignoriert jedes Skript, das Ressourcen der Domain über HTTP laden will. HSTS ist ein Teil eines großen Bündels an Sicherheitsmaßnahmen, das Sie für Ihren Webserver oder Webhosting-Dienst nutzen können.
 
 Um HSTS zu aktivieren gibt es mehrere Möglichkeiten. Die Aufzählung erfolgt in der Reihenfolge der Prio: 
```

Comment:
Hier würde ich nur die von REDAXO bereitgestellten Möglichkeiten im Setup und in der config.yml listen und auf den guten Rat des Hosters verweisen.